### PR TITLE
Update REVIEWERS.md

### PR DESCRIPTION
**Pull request by michaeljs1990**
Added myself.... and updated the formatting so you have less eye bleeding when looking at it.

### CODE REVIEW

**Comment by bcoca** [comment]
cc @gregdek

File: REVIEWERS.md [hunk 1]
```diff
@@ -6,154 +6,154 @@ Two +1 votes by any of these module reviewers on a new module pull request will
...
+-  Philippe Makowski [@pmakowski](https://www.github.com/pmakowski) 
+-  Phillip Gentry, CX Inc [@pcgentry](https://www.github.com/pcgentry) 
+-  Quentin Stafford-Fraser [@quentinsf](https://www.github.com/quentinsf) 
+-  Ramon de la Fuente [@ramondelafuente](https://www.github.com/ramondelafuente) 
+-  Raul Melo [@melodous](https://www.github.com/melodous)  
+-  Ravi Bhure [@ravibhure](https://www.github.com/ravibhure) 
+-  René Moser [@resmo](https://www.github.com/resmo) 
+-  Richard Hoop [@rhoop](https://www.github.com/rhoop) 
+-  Richard Isaacson [@risaacson](https://www.github.com/)  
+-  Rick Mendes [@rickmendes](https://www.github.com/risaacson)
```
Comment (ghost) [inline REVIEWERS.md]: While you are doing this, can you fix my github link?

It should be https://github.com/rickmendes/. 

I didn't have proper permissions to push the change myself to your PR. Thanks.

Comment (michaeljs1990) [inline REVIEWERS.md]: will do sorry about that.

Comment (michaeljs1990) [inline REVIEWERS.md]: fixed now.

Comment (ghost) [inline REVIEWERS.md]: Thank you!

**Comment by gregdek** [comment]
Thanks @michaeljs1990!

**Comment by michaeljs1990** [comment]
Thanks for the quick merge :+1:

**Pull request merged**